pypesto/ensemble/ensemble.py

```diff
@@ -1111,29 +1111,25 @@ def entries_per_start(
     ens_ind = [np.flatnonzero(fval <= cutoff) for fval in fval_traces]
 
     # count the number of candidates per start
-    n_per_start = np.array([len(start) for start in ens_ind])
+    n_theo = np.array([len(start) for start in ens_ind])
```

Comment:
This and the previous line look like they could be combined into

```suggestion
    n_theo = np.array([(fval <= cutoff).sum() for fval in fval_traces)])
```